### ISSUE TITLE
Plugin not yet activated

### ISSUE THREAD

**Issue by ezimuel** [klesun/deep-assoc-completion-vscode#7]
I'm trying to use this plugin in vscode but the runtime status said "Not yet activated" (see screenshot). I tried to use it using the example reported in the details but nothing appen when I press `Ctrl + Space`. I also disabled the `php.suggest.basic`.

What can I do for activating the plugin? Thanks!

![image](https://user-images.githubusercontent.com/475967/139027044-8a837554-ce9f-489e-beca-aec2377df1aa.png)

**Comment by klesun** [comment]
Hm, I'm getting same "Not yet activated" message. I wonder if there was some [breaking change](https://stackoverflow.com/questions/67992050/vs-code-not-activating-extensions) in a recent vscode version...

**Comment by ezimuel** [comment]
@klesun I really would like to contribute to this project since it will be very useful for [elastic/elasticsearch-php](https://github.com/elastic/elasticsearch-php) project. Here we have all the API exposed as function with an associative array as parameter (see [here](https://github.com/elastic/elasticsearch-php/blob/master/src/Elasticsearch/Client.php#L307-L326) as example). I'm rewriting the phpDoc section to be compatible with [PHPstan array shapes](https://phpstan.org/writing-php-code/phpdoc-types#array-shapes), as follows:
```php
/**
 * Allows to perform multiple index/update/delete operations in a single request.
 *
 * @see https://www.elastic.co/guide/en/elasticsearch/reference/master/docs-bulk.html
 *
 * @param array{
 *     index: string, // (REQUIRED) Default index for items which don't provide one
 *     type: string, //  Default document type for items which don't provide one
 *     wait_for_active_shards: string, // Sets the number of shard copies that must be active before proceeding with the bulk operation. Defaults to 1, meaning the primary shard only. Set to `all` for all shard copies, otherwise set to any non-negative value less than or equal to the total number of copies for the shard (number of replicas + 1)
 *     refresh: enum, // If `true` then refresh the affected shards to make this operation visible to search, if `wait_for` then wait for a refresh to make this operation visible to search, if `false` (the default) then do nothing with refreshes.
 *     routing: string, // Specific routing value
 *     timeout: time, // Explicit operation timeout
 *     type: string, // Default document type for items which don't provide one
 *     _source: list, // True or false to return the _source field or not, or default list of fields to return, can be overridden on each sub-request
 *     _source_excludes: list, // Default list of fields to exclude from the returned _source field, can be overridden on each sub-request
 *     _source_includes: list, // Default list of fields to extract and return from the _source field, can be overridden on each sub-request
 *     pipeline: string, // The pipeline id to preprocess incoming documents with
 *     require_alias: boolean, // Sets require_alias for all incoming documents. Defaults to unset (false)
 *     body: array, // (REQUIRED) The operation definition and data (action-data pairs), separated by newlines
 * } $params
 */
```

I already tested your [deep-assoc-completion](https://github.com/klesun/deep-assoc-completion) for PhpStorm and it works perfectly! :heart:

I'll try to allocate some times for studying your code base and hopefully I'll send some PRs :crossed_fingers:

**Comment by klesun** [comment]
That would be very nice =D
I suspect the issue is somewhere in the initialization process, if I remember correctly, [I'm doing](https://github.com/klesun/deep-assoc-completion-vscode/blob/master/src/main.js) `npm install` and using `ts-node` on client's machine, which is somewhat unorthodox, but worked back then.
If I'll get some time, I'd like to try debug this issue as well, possibly may get to it on the weekend...

**Comment by klesun** [comment]
Likely related: https://issueexplorer.com/issue/ansible/vscode-ansible/94

**Comment by klesun** [comment]
Sorry, did not get the time, and probably won't get it anytime soon, too much IRL stuff ;c

**Comment by ezimuel** [comment]
No problem, I'm starting getting confidence in the code.

**Comment by ciricode** [comment]
Is there any update for it? =(

**Comment by klesun** [comment]
Seems to work in vscode [v1.44](https://code.visualstudio.com/updates/v1_44) at least (the vscode version back from 2020)

![image](https://user-images.githubusercontent.com/5202330/158956519-fab0b296-68eb-4d14-80b0-3ce7e4fcef3a.png)

One could use a binary search to find in what version did it get broken =D. That will probably help in nailing down the cause of the problem.

**Comment by klesun** [comment]
With the power of binary search I found out that the vscode version where plugin stopped working was [v1.53](https://code.visualstudio.com/updates/v1_53). In [v1.52](https://code.visualstudio.com/updates/v1_52) it still worked. Will read the patchnotes...

Probably related to this: https://github.com/microsoft/vscode/issues/116219

**Comment by klesun** [comment]
Released a fix: `v0.8.1`

It now works in the latest vscode!

![image](https://user-images.githubusercontent.com/5202330/159175767-05e977a8-265a-47aa-aaeb-ff4e12142720.png)

**Comment by ezimuel** [comment]
@klesun awesome, thanks!
